# This represents new buoyancy parameters for objects
Header header

string model_name
string link_name

float32 compensation
float32 limit

float32 x_damping
float32 y_damping
float32 z_damping

bool has_origin

float32 x_origin
float32 y_origin
float32 z_origin
